cypress/integration/inventory/actions-menu.spec.js

```diff
@@ -0,0 +1,18 @@
+import TopMenu from '../../support/fragments/topMenu';
+import Actions from '../../support/fragments/inventory/actions';
+
+describe('ui-inventory: actions', () => {
+  beforeEach('navigates to Actions', () => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.visit(TopMenu.inventoryPath);
+  });
+
+  it('C196752 verifies action menu options before any search is conducted', () => {
+    Actions.openActions();
+
+    Actions.optionIsDisabled(Actions.saveUUIDOption, true);
```

Comment:
you can specify list of disabled options and apply foreach to run optionIsDisabled.
Additionally, looks like second parameters  - excess. as I see from name - method checks disabled options.

Reply:
> you can specify list of disabled options and apply foreach to run optionIsDisabled.
Yes, good idea

> Additionally, looks like second parameters - excess. as I see from name - method checks disabled options.
mm, what if in other test I need check that another option is enabled? I want to have one assert function for this

Reply:
renaming can help in this case too

Reply:
for example - checkOptionAvailability(). Or you can one internal method with second parameter + 2 public methods without extra parameter.

Reply:
Why I can't have 1 method with 2 parametrs for every check enabled/disable state? Why I nedd 3 extra methods instead of 1?